docs/ingress.md

```diff
@@ -0,0 +1,72 @@
+# Ingress
+
+## Executive summary
+
+In several cases, our applications have a `frontend` and a `backend`, and
+sometimes, the `backend` can communicate with a `database`. In this file, you
```

Comment:
database or object storage?

also for our specific AI products, we have AI models

Reply:
Done